policy/modules/contrib/screen.te

```diff
@@ -96,3 +97,7 @@ userdom_use_user_terminals(screen_domain)
 userdom_create_user_pty(screen_domain)
 userdom_setattr_user_ptys(screen_domain)
 userdom_setattr_user_ttys(screen_domain)
+
+tunable_policy(`screen_allow_session_sharing',`
+allow screen_domain self:capability { setgid setuid };
```

Comment:
Yes that is the gist, you could also add the fsetid here (i think the sys_tty_config can probably safely be dontaudited)

Reply:
So fsetid is somehow relates with session sharing? And about the capability of sys_tty_config, we don't know impact of this change, when we will dontaudit this capability. Sorry for late reply, but I have had a lot of work to do.

Reply:
Yes, capabilties require root, and root is gained with setuid. The fsetid is used to set proper ownership on the sock files used to connect to sessions

Reply:
Thank you for explanation.